src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const visibleTabId =
+    tabs.findIndex(tab => tab.id === selectedTabId) > -1
+      ? selectedTabId
+      : tabs[0].id;
```

Comment:
```suggestion
  const visibleTabId = tabs.find(tab => tab.id === selectedTabId).id || tabs[0].id
```

we can simplify it a bit